README.md

```diff
@@ -34,8 +40,11 @@ For information relating to the development of Rust applications on ESP devices,
 
 For information about the HAL and how to use it in your own projects, please refer to the [documentation].
 
+When browsing the examples, we recommend viewing the tag for the `esp-hal` release you are using to ensure compatibility, e.g.) [v0.22.0], as the `main` branch is used for development and APIs may have changed in the meantime.
+
 [The Rust on ESP Book]: https://esp-rs.github.io/book/
 [documentation]: https://docs.esp-rs.org/esp-hal/
+[v0.22.0]: https://github.com/esp-rs/esp-hal/tree/v0.22.0/examples
```

Comment:
We should add to our release process a reminder to update this link

Reply:
Done!